src/event-publisher.ts

```diff
@@ -16,16 +16,16 @@ export class EventPublisher<EventBase extends IEvent = IEvent> {
   ): T {
     const eventBus = this.eventBus;
     return class extends metatype {
-      publish(event: EventBase) {
-        eventBus.publish(event);
+      publishAll(events: EventBase[]) {
+        eventBus.publishAll(events);
```

Comment:
Shouldn't we leave the `publish()` implementation and just add the `publishAll()`?

Reply:
Yes, I suppose that does make a bit more sense.